readme.md

```diff
@@ -147,6 +168,24 @@
 - [WorldGuard](https://github.com/EngineHub/WorldGuard) - WorldGuard lets you and players guard areas of land against griefers and undesirables, as well as tweak and disable various gameplay features of Minecraft.
 - [CraftBook](https://github.com/EngineHub/CraftBook) - CraftBook lets you create magically extending bridges, compact Redstone circuits, complex Minecart mechanics, and much more — all without a client mod and fully customizable by the server.
 
+## Resource Packs
+
+- [BDCraft](https://bdcraft.net/) - Various comic-style resource and sound packs, even for many mods, Bedrock Edition, and Terraria!
+- [LB Photo Realism Reload](https://www.curseforge.com/minecraft/texture-packs/lb-photo-realism-reload) - Successor of the LB Photo Realism texture pack making Minecraft look more realistic.
+- [Default Dark Mode](https://www.curseforge.com/minecraft/texture-packs/default-dark-mode) - Dark Mode for the default Minecraft UIs!
+- [Cyber Optics](https://www.planetminecraft.com/texture-pack/cyber-optics-v10/) - Sci-Fi resource pack.
+- [Soartex](https://soartex.net/) - Smooth resource pack with mod support.
+- [Faithful](https://faithful.team/) - Smooth textures for a furbished default look.
+
+## Shader Packs
+
+- [Continuum](https://continuum.graphics/) - A realistic shader pack.
+- [Chocapic13](https://www.curseforge.com/minecraft/customization/chocapic13-shaders) - A realistic and configurable shader pack.
+- [Sonic Ether](https://www.sonicether.com/) - Pathracing and other realistic shader packs.
+- [Sildur](https://sildurs-shaders.github.io/) - Shader packs even implementing Minecraft 1.16's built-in API.
+- [Potato](https://rre36.com/potato-shader) - A very lightweight shader pack for... well... potato PCs.
```

Comment:
What's the `...`?

Reply:
It's a little joke if that's okay. I figured, one could make this list a little bit more interesting by adding stuff like this (It's the only one right now)

Reply:
Joke is okay, but I'm afraid whether others can understand this description or not.

Reply:
Is that better?

Reply:
:+1: